Console/Command/MigrationShell.php

```diff
@@ -118,6 +121,32 @@ public function startup() {
 	}
 
 /**
+ * Get connection names list
+ * @return Array
+ */
+	protected function _connectionNamesEnum() {
+		return array_keys($this->ConnectionManager->enumConnectionObjects());
```

Comment:
enumConnectionObjects is a static function https://github.com/cakephp/cakephp/blob/2.6/lib/Cake/Model/ConnectionManager.php#L192